### PR TITLE
Fix English in README.md and some tiny gradle rearrangements

### PR DESCRIPTION
**Pull request by TacoTheDank**
- Group and reorder dependencies
- Note that ```de.measite.minidns:minidns-hla``` has been renamed to ```org.minidns:minidns-hla``` in later versions
- ```org.whispersystems:signal-protocol-java``` 2.6.2 -> 2.7.0
- Encoding set to UTF-8
- Put minifyEnabled before shrinkResources, as minifyEnabled occurs first
- Fix lots of English grammar, update badges (and add F-Droid badge), and update links

### CODE REVIEW

**Comment by licaon-kter** [comment]
F-Droid badge, remove it, there's a reason there aro links only to the bought versions

The motto "last word..." doesn't need a dot at the end

**Comment by TacoTheDank** [comment]
lol alright then

File: build.gradle [hunk 1]
```diff
@@ -31,39 +32,48 @@ ext {
...
-    implementation "com.android.support:design:$supportLibVersion"
-    compatImplementation "com.android.support:support-emoji-appcompat:$supportLibVersion"
-    conversationsFreeCompatImplementation "com.android.support:support-emoji-bundled:$supportLibVersion"
-    quicksyFreeCompatImplementation "com.android.support:support-emoji-bundled:$supportLibVersion"
-    implementation 'org.bouncycastle:bcmail-jdk15on:1.58'
-    implementation 'com.google.zxing:core:3.3.3'
+
+    // Other
     implementation 'de.measite.minidns:minidns-hla:0.2.4'
+    //implementation 'org.minidns:minidns-hla:0.3.4' //0.4.0-alpha3
```
Comment (licaon-kter) [inline build.gradle]: What's the point of this line?

File: build.gradle [hunk 2]
```diff
@@ -31,39 +32,48 @@ ext {
...
-    implementation "com.wefika:flowlayout:0.4.1"
+    implementation 'com.wefika:flowlayout:0.4.1'
     implementation 'net.ypresto.androidtranscoder:android-transcoder:0.3.0'
-    implementation project(':libs:xmpp-addr')
     implementation 'org.osmdroid:osmdroid-android:6.0.3'
     implementation 'org.hsluv:hsluv:0.2'
-    implementation 'org.conscrypt:conscrypt-android:1.3.0'
+    implementation 'org.conscrypt:conscrypt-android:1.3.0' //1.4.2
     implementation 'me.drakeet.support:toastcompat:1.1.0'
+    implementation 'org.sufficientlysecure:openpgp-api:10.0' //12.0
```
Comment (licaon-kter) [inline build.gradle]: If updated...
```
> Task :compileConversationsFreeCompatReleaseJavaWithJavac
/home/android/Ctest0114/src/main/java/eu/siacs/conversations/crypto/PgpEngine.java:293: error: reference to executeApi is ambiguous
                Intent result = api.executeApi(params, null, null);
                                   ^
  both method executeApi(Intent,InputStream,OutputStream) in OpenPgpApi and method executeApi(Intent,ParcelFileDescriptor,OutputStream) in OpenPgpApi match
1 error
```

**Comment by iNPUTmice** [comment]
If you are going to bump library versions you should at least make sure that it compiles. I’ll happily merge the changes to README; but a PR that messes with `build.gradle` and then doesn’t even compile makes me suspicious.

**Comment by TacoTheDank** [comment]
ah, alright

**Comment by TacoTheDank** [comment]
@licaon-kter I'm just fixing up the lines that are already there lol, I have no part in how exactly they decide to word the sentences

File: build.gradle (hunk 2 above)
Comment (TacoTheDank) [reply build.gradle]: I didn't update this though, unless that's not what you're getting at

**Comment by licaon-kter** [comment]
Maybe you need to rebase?

**Comment by TacoTheDank** [comment]
hmm maybe

**Pull request closed without merging**